tests/driver_at24cxxx/main.c

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (C) 2019 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief    AT24CXXX test application
+ *
+ * Tested with bluepill and AT24C256
+ *
+ * @}
+ */
+
+#include <errno.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "xtimer.h"
+#include "board.h"
+
+#include "at24cxxx.h"
+
+#define I2C_DEVICE           I2C_DEV(0)
+
+#define WRITE_BYTE_POSITION  (12U)
+#define WRITE_BYTE_CHARACTER 'A'
+
+#define WRITE_POSITION       (444U)
+#define WRITE_CHARACTERS     { 'B', 'E', 'E', 'R', '4', 'F', 'R', 'E', 'E', \
+                               '\0' }
+
+#define SET_POSITION         (900U)
+#define SET_CHARACTER        'G'
+#define SET_LEN              (20U)
+
+int main(void)
+{
+    puts("Starting tests for module at24cxxx");
+    xtimer_init();
+    i2c_init(I2C_DEVICE);
```

Comment:
All available buses are initialized on `periph_init()` during boot up. Initializing the device twice could cause issues.

Reply:
Good to know, I was wondering why no driver was calling `i2c_init()`.